CHANGELOG.md

```diff
@@ -22,6 +26,10 @@ You may also find the [Update Guide](UPDATING.md) useful.
 - Linux, NetBSD, Solaris: read in blocking mode on first use in `fill_bytes`. (#484)
 - Fuchsia, CloudABI: fix compilation (broken in Rand 0.5). (#484)
 
+### Other changes
+- References to a `CryptoRng` now also implement `CryptoRng`. (#470)
```

Comment:
This change was in rand_core, not in rand. Maybe best to only add it in a changelog there?

Reply:
Whoops. Then we also need a new `rand_core` release (at some point).

Reply:
We don't really have other changes for rand_core in the pipeline at the moment I think. Maybe release rand_core 0.2.1 at the same time as rand 0.5.1?

Reply:
I suppose I can release that tomorrow too.